include/z64dma.h

```diff
@@ -21,4 +22,28 @@ typedef struct {
     /* 0x0C */ uintptr_t romEnd;
 } DmaEntry;
 
+extern DmaEntry gDmaDataTable[];
+
+extern u32 gDmaMgrVerbose;
+extern size_t gDmaMgrDmaBuffSize;
+
+#define DMAMGR_DEFAULT_BUFSIZE ALIGN16(0x2000)
+
+// Standard DMA Requests
+
+s32 DmaMgr_SyncDmaRequest(void* ram, uintptr_t vrom, size_t size);
+s32 DmaMgr_SyncDmaRequestDebug(void* ram, uintptr_t vrom, size_t size, const char* file, s32 line);
+s32 DmaMgr_AsyncDmaRequest(DmaRequest* req, void* ram, uintptr_t vrom, size_t size, u32 unk5, OSMesgQueue* queue,
+                           OSMesg msg, const char* file, s32 line);
+
```

Comment:
my gut feeling is that these names feel a little "backward" to me. 
It fees to me like the name should be structured like DmaMgr_ThingType, so `DmaRequestAsync`/`DmaRequestSync`
And I also kinda feel like "Dma" being there is a bit redundant with the system name already specifying it (`DmaMgr_RequestAsync`/`DmaMgr_RequestSync`/`DmaMgr_RequestSyncDebug`)

not 100% sold on any of these names, but just throwing my thoughts out there

Reply:
I did some research on conventions out there for sync vs async vs (nothing), prefix vs suffix, and there is no big common pattern like `Impl` can sometimes be

https://www.reddit.com/r/csharp/comments/s6w5fv/why_do_people_add_async_to_method_names_in_new/ summarizes it best:
> Ehh. Ask 100 devs and you'll probably get 200 different answers.

I personally ... don't have an opinion

-------

As for "Dma" I must say I could see it omitted indeed, `DmaMgr_SyncRequest` looks fine, but I'm also fine if it stays